pkg/apis/metal3/v1alpha1/baremetalhost_types.go

```diff
@@ -104,6 +104,21 @@ type BMCDetails struct {
 	CredentialsName string `json:"credentialsName"`
 }
 
+// BIOSConfig contains the configuration that you want to configure BIOS settings in Bare metal server
+type BIOSConfig struct {
+
+	// Select the boot mode of the system. Allowed values are Uefi, LegacyBios
+	// +kubebuilder:validation:Enum=Uefi,LegacyBios
```

Comment:
I think we want the enum values to be all lower case.

"LegacyBios" feels redundant. Is there another type of legacy boot interface? Should we just call it "bios"?

Reply:
>I think we want the enum values to be all lower case.

Done. I will start the enum with lower case. 

> Should we just call it "bios"?

Hmm, I think `bios` may make the user confuse. `legacy` is better, I think.